catalog/resourcemanagement/resourcemanagement-usage-plugin/src/main/java/org/codice/ddf/resourcemanagement/usage/ResourceUsagePlugin.java

```diff
@@ -46,8 +51,8 @@ public ResourceRequest process(ResourceRequest input)
             throws PluginExecutionException, StopProcessingException {
         long resourceSize = 0L;
 
-        if (input != null) {
-
+        if (input != null && ((!monitorLocalSources && Requests.isEnterprise(input))
```

Comment:
Isn't `(monitorLocalSources || Requests.isEnterprise(input))` equivalent to this?